cms/jinja2/templates/components/streamfield/topic_headline_figure_block.html

```diff
@@ -1,19 +1,22 @@
 {% from "components/card/_macro.njk" import onsCard %}
 
+{% set error_text = _("Error") %}
+{% set error_supporting_text = _("Error retrieving figure") %}
+
 {# fmt:off #}
     {{
```

Comment:
would it not make more sense to wrap this in an `if figure and figure.title` and only display if the figure data is available?